test/linters/params/rustfmt.js

```diff
@@ -0,0 +1,72 @@
+const { join } = require("path");
+
+const RustFmt = require("../../../src/linters/rustfmt");
+
+const testName = "rustfmt";
+const linter = RustFmt;
+const commandPrefix = "";
+const args = " -- --color=never ";
```

Comment:
Would it be better to add this as a default arg in `rustfmt_args`?

Reply:
Good catch!
I was only playing with ubuntu builds and didn't think through the full impact of the windows test fix.
[31f7e4b3](https://github.com/wearerequired/lint-action/pull/585/commits/31f7e4be1cf236dc5e69ee854d57b7253e236367) defaults to disabling colors in `action.yml`

it does look pretty noisy with color left to the default (from https://github.com/nick-mobilecoin/rust-actions/actions/runs/3997571937/jobs/6859115018)
![image](https://user-images.githubusercontent.com/101757619/214334974-b4bcc510-1a41-4a39-80ca-c47b2fa2a0e9.png)

Passing the flag results in (from https://github.com/nick-mobilecoin/rust-actions/actions/runs/3997616381/jobs/6859307621)
![image](https://user-images.githubusercontent.com/101757619/214335119-89cfde32-885e-49a5-9d19-adb9c0e0fb2b.png)